cnvlib/coverage.py

```diff
@@ -104,15 +112,61 @@ def filter_read(read):
     return count, depth
 
 
-def interval_coverages_pileup(bed_fname, bam_fname, min_mapq):
+def rm(path):
+    try:
+        os.unlink(path)
+    except OSError:
+        pass
+
+def to_chunks(bed_fname, chunk_size=5000):
```

Comment:
On my data, 5000 lines takes about 1 minute per process so it stays nicely parallel. I was thinking that the bigger the chunk, the more risk there is of not keeping the cores busy. Though certainly something like 20,000 should be fine.

I'm happy to use your machinery here if you have something else in mind.